pkg/registries/filter_test.go

```diff
@@ -0,0 +1,209 @@
+package registries
+
+import (
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+
+	yaml "gopkg.in/yaml.v1"
+
+	ft "github.com/openshift/ansible-service-broker/pkg/fusortest"
+)
+
+const testWhitelistFile = "whitelist.yaml"
+const testBlacklistFile = "blacklist.yaml"
+const testBlacklistOverrideFile = "blacklist_override.yaml"
+
+func testGetRegexFromFile(file string) []string {
```

Comment:
Does this actually run? I always thought they had to be `TestXYZ`. I learned something new today.